src/clj/puppetlabs/services/jruby_pool_manager/jruby_schemas.clj

```diff
@@ -59,14 +59,17 @@
         values are `:jit`, `:force`, and `:off`.  Defaults to `:off`.
 
     * :max-active-instances - The maximum number of JRubyInstances that
-        will be pooled."
+        will be pooled.
+
+    * :environment-vars - The whitelisted environment variables to be passed"
```

Comment:
I'm interested in making it clear that the values will be passed in via the map and not from the outer environment.